src/Arguments.ts

```diff
@@ -0,0 +1,4 @@
+export type Arguments = {
+  xmlFile: string;
+  xsdFile: string | undefined;
+};
```

Comment:
This is good, and reminds me that when we merge teams, we should talk a little about TypeScript vs. C# (I assume the AMT team is heavily .NET), in particular how TypeScript has separate aspects of "type" and "module" from "class" in ways where "class" is often heavier-weight than needed.